knowledge/Users/klawrenc/.local/share/instructlab/taxonomy/knowledge/films/history/qna.yaml

```diff
@@ -0,0 +1,135 @@
+created_by: klawrenc
+version: 3
+domain: film history
+document_outline: >-
+  reference for films over the years, how it was made and the general history of
+  cinema
+seed_examples:
+  - context: This model is a resource for the history of film
```

Comment:
The `context` field should be content from the source document referenced in the attribution file. The content you pull from that document for this field should include enough information for a random reader to answer the questions you pose in the `questions_and_answers` section, as if someone is reading for a test. Note that the `context` field can have up to 500 words.

All of these `context` fields need more information, and it needs to come from a source document.

Reply:
Also, for reference, the model will not answer exactly as you have in the seed examples. It may use different wording or phrases and may structure answers differently. The data, though, will always be the same. So don't worry about exact answer wording.